lib/zod/schema/LoginSchema.ts

```diff
@@ -1,8 +1,8 @@
 import { z } from "zod";
 
-export type LoginSchema = z.infer<typeof LoginSchema>;
+export type Login = z.infer<typeof loginSchema>;
```

Comment:
요런 느낌으로 되면 어떨까 싶네요!
form 데이터를 가리키는 타입인 것 같은데 Login이라고만 표현되면 어디에 쓰일지 알기 어려운 것 같아요~
```suggestion
export type LoginForm = z.infer<typeof loginSchema>;
```